devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
I see, I thought the API was for both users and us, so I thought we should document everything. Do you want to not include `function_manager` and `cgen_wrapper`  and the 2 classes then?